ghost-cat-app/src/app/View/image-thumbnails-page/image-thumbnails-page.component.html

```diff
@@ -24,18 +24,11 @@
       [boundingBoxes]="items[currIndex].data.boundingBoxes">
     </app-sidebar>
 
-    <div class="topnav">
```

Comment:
I made some changes to the main branch a while ago that I thought would fix the problem with the icons... It looks like you were working with an older version of main so you can look and see which version you'd like to keep.